src/zappy_gui_src/Core/SFML/Map/Map.cpp

```diff
@@ -99,6 +102,10 @@ void Map::display()
         }
         _tile[i]->setColor(sf::Color(255, 255, 255));
         _window->draw(_tile[i]->getShape());
+        if (_tile[i]->getPlayers().size()) {
+            _playerRepresentation.setPosition({_tile[i]->getIsoPosition().left + _tile[i]->getIsoPosition().width / 2, _tile[i]->getIsoPosition().top + _tile[i]->getIsoPosition().height / 2});
+            _window->draw(_playerRepresentation);
+        }
```

Comment:
It could be split in different static function of display ( player, map, etc...) to make it easier to read but the code LGTM